### ISSUE TITLE
Feature Request: filter by service body/meeting format

### ISSUE THREAD

**Issue by EzraKuhr** [bmlt-enabled/bmlt-wordpress-satellite-plugin#12]
I would like to see the satellite plugin be able to filter all views by service body or meeting formats.

**Comment by radius314** [comment]
This is already possible with shortcodes.  Were you looking for something more specific?

**Comment by pjaudiomv** [comment]
I don't believe that is true for basic bmlt shortcode that displays map `[[bmlt]]` and although he didn't specify, that is what he is requesting.

**Comment by radius314** [comment]
I have tried before to exclude certain formats and it worked.

**Comment by pjaudiomv** [comment]
awesome!! can you provide an example of short code usage.

**Comment by pjaudiomv** [comment]
this page https://bmlt.app/satellites/cms-plugins/shortcodes/ states that the only parameter allowed for BMLT, BMLT_MAP, BMLT_MOBILE and SIMPLE_SEARCH_LIST is the settings ID

All keys can have parameters. These are supplied immediately after the key, in parentheses (). In the case of the BMLT, BMLT_MAP, BMLT_MOBILE and SIMPLE_SEARCH_LIST, the only parameter allowed is the settings ID for the setting that will be used by that key. In the case of the BMLT_SIMPLE code, you would place the “simple” API parameters (Described in this page). You can also add the settings ID, by separating them with “##-##“.

**Comment by pjaudiomv** [comment]
If using the Advanced Map Initial Search Type, you can specify to only search within a certain service body or format. but the request here is to not have those come up at all as available option.

**Comment by EzraKuhr** [comment]
Thank you Patrick for articulating my request and the research for it.

**Comment by pjaudiomv** [comment]
this would most likely need to be done in the [bmlt-satellite-base-class](https://github.com/bmlt-enabled/bmlt-satellite-base-class/)

**Comment by radius314** [comment]
Ok thanks

**Comment by pjaudiomv** [comment]
https://github.com/bmlt-enabled/bmlt-satellite-base-class/blob/381e2f44f6d731838f3434744ca2d899f0c84ee0/bmlt-cms-satellite-plugin.php#L1879-L1910

**Comment by radius314** [comment]
I would be great to do this as a custom query.
